pom.xml

```diff
@@ -47,10 +47,16 @@
             <artifactId>jakarta.json.bind-api</artifactId>
             <version>${jakarta.json.bind.version}</version>
         </dependency>
+        <dependency>
+            <groupId>jakarta.json</groupId>
+            <artifactId>jakarta.json-api</artifactId>
+            <version>${jakarta.json.version}</version>
+        </dependency>
         <dependency>
             <groupId>org.glassfish</groupId>
             <artifactId>jakarta.json</artifactId>
             <version>${jakarta.json.version}</version>
+            <classifier>module</classifier>
```

Comment:
will this continue to be compatible with Java 8 users?

Reply:
It should be since module-info is ignored in Java 8 and this is just variation of the jar which does not contain API. (That is why I have to added it explicitly above this dependency.)